pkg/storage/s3storage.go

```diff
@@ -347,7 +347,7 @@ func (s3 *S3Storage) FullPath(name string) string {
 func (s3 *S3Storage) Read(name string) ([]byte, error) {
 	name = trimLeading(name)
 
-	log.Debugf("S3Storage::Read(%s)", name)
+	log.Tracef("S3Storage::Read(%s)", name)
```

Comment:
I seem to recall this being useful for debugging, but maybe we have another higher-level log in Aggregator ingestion which outputs the file name also?

Reply:
I believe it was most useful for ETL Utils. However we can add in some higher-level INF or DBG logging if needed.